src/SamV71Core/SamV71Core.c

```diff
@@ -144,6 +145,71 @@ void SamV71Core_Init(void)
 {
 	Pmc_init(&pmc, Pmc_getDeviceRegisterStartAddress());
 
+	const Pmc_Config pmcConfig = {
```

Comment:
If you are working with clock configuration, be mindful of requirements:
MBEP-RT-FUN-710	TASTE Runtime shall be able to configure CPU core frequency to match the target for the runtime.
MBEP-RT-FUN-720	It shall be possible to disable the CPU core frequency configuration.

Consult with SDD on RT_RTOS_NO_INIT